src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class FruitTransactionTest {
+
```

Comment:
For the future, remove the first empty line in all classes/interfaces/methods where it is present